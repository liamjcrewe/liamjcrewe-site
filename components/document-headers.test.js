import { render } from '@testing-library/react'

import DocumentHeaders from './document-headers'

describe('DocumentHeaders component', () => {
  it('renders and matches snapshot', () => {
    const { container } = render(<DocumentHeaders />, {
      container: document.createElement('head')
    })

    expect(container).toMatchSnapshot()
  })
})
