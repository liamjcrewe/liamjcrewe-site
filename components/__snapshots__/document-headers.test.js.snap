// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DocumentHeaders component renders and matches snapshot 1`] = `
<head>
  <title>
    Liam Crewe | Software Developer
  </title>
</head>
`;
